extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -0,0 +1,310 @@
+			'buttonLink' => $this->getButtonLink(),
+			'details' => $this->getDetailsList(),
+			'contentFooterMessages' => [
+				$this->getCommunityFooterMessage()
+			],
+			'hasContentFooterMessages' => true
+		] );
+	}
+
+	public function assertCanEmail() {
+		parent::assertCanEmail();
+		$this->assertSubscribedToCompleteDigest();
+	}
+
+	/**
+	 * Asserts that the founder is subscribed to the complete digest.
+	 *
+	 * @throws \Email\Check
+	 */
+	protected function assertSubscribedToCompleteDigest() {
+		if ( !$this->targetUser->getBoolOption( 'founderemails-complete-digest-' . $this->wikiId ) ) {
+			throw new Check( 'Founder is not subscribed to complete digest.' );
+		}
+	}
+
+	protected function assertValidParams() {
+		parent::assertValidParams();
+		$this->assertNewUsersSet();
+		$this->assertPageEditsSet();
+	}
+
+	/**
+	 * Asserts that the `newUsers` value is passed
+	 *
+	 * @throws \Email\Check
+	 */
+	protected function assertNewUsersSet() {
+		if ( empty( $this->newUsers ) && $this->newUsers !== '0' ) {
+			throw new Check( 'Invalid value passed for `newUsers`' );
+		}
+	}
+
+	/**
+	 * Asserts that the `pageEdits` value is passed
+	 *
+	 * @throws \Email\Check
+	 */
+	protected function assertPageEditsSet() {
+		if ( empty( $this->pageEdits ) && $this->pageEdits !== '0' ) {
+			throw new Check( 'Invalid value passed for `pageEdits`' );
+		}
+	}
+
+	/**
+	 * Returns list of details for the digest
+	 *
+	 * @return array
+	 */
+	protected function getDetailsList() {
+		return [
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'Page-views.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-views-header', $this->pageViews )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-views-description-1' )->text()
+			],
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'Number-of-edits.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-edits-header', $this->pageEdits )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-edits-description' )->text()
+			],
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'New-users.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-users-header', $this->newUsers )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-users-description' )->text()
+			] 
+		];
+	}
+
+	public function getButtonText() {
+		return $this->getMessage( 'emailext-founder-activity-digest-link-label' )->text();
+	}
+
+	public function getButtonLink() {
+		return \GlobalTitle::newFromText( 'WikiActivity', NS_SPECIAL, $this->wikiId )->getFullURL();
+	}
+
+	protected function getCommunityFooterMessage() {
+		return $this->getMessage(
+			'emailext-founder-activity-digest-footer',
+			\GlobalTitle::newMainPage( \Wikia::COMMUNITY_WIKI_ID )->getFullURL()
```

Comment:
Yeah, @jsutterfield suggested that the Community URL should be hardcoded in messages, as it was done for another email, and translated for each language. So this parameter can just be removed.